pegasus/helpers/hoc_helpers.rb

```diff
@@ -98,11 +99,14 @@ def complete_tutorial(tutorial={})
         finished_ip: request.ip,
       )
     else
-      row = create_session_row(
+      # Use weight of 0 for unsampled session so we don't double count.
```

Comment:
Expanding the comment would be helpful (at least to me).
# Use weight of 0...
# Otherwise, use a weight of 1 as the user was not involved in a sampling decision. This can happen when...